tests/manage/mcg/test_namespace_rpc.py

```diff
@@ -36,203 +40,10 @@ class TestNamespace(MCGTest):
-        bucketclass_dict,
-    ):
-        """
-        Test writing to AWS and reading from an ns bucket
-        """
-
-        ns_bucket = bucket_factory(
-            amount=1,
-            interface=bucketclass_dict["interface"],
-            bucketclass=bucketclass_dict,
-        )[0]
-
-        s3_creds = {
-            "access_key_id": cld_mgr.aws_client.access_key,
-            "access_key": cld_mgr.aws_client.secret_key,
-            "endpoint": constants.MCG_NS_AWS_ENDPOINT,
-            "region": self.DEFAULT_REGION,
-        }
-        aws_target_bucket = ns_bucket.bucketclass.namespacestores[0].uls_name
-
-        # Upload files directly to AWS
-        self.write_files_to_pod_and_upload(
-            mcg_obj,
-            awscli_pod,
-            bucket_to_write=aws_target_bucket,
-            amount=3,
-            s3_creds=s3_creds,
-        )
-        # Read files from ns bucket
-        self.download_files(mcg_obj, awscli_pod, bucket_to_read=ns_bucket.name)
-
-        # Compare between uploaded files and downloaded files
-        assert self.compare_dirs(awscli_pod, amount=3)
-
-    @skipif_ocs_version("<4.7")
-    @tier1
-    @pytest.mark.parametrize(
-        argnames=["bucketclass_dict"],
-        argvalues=[
-            pytest.param(
-                {
-                    "interface": "OC",
-                    "namespace_policy_dict": {
-                        "type": "Single",
-                        "namespacestore_dict": {"aws": [(1, "eu-central-1")]},
-                    },
-                }
-            ),
-        ],
-    )
-    def test_write_to_ns_read_from_aws_crd(
-        self,
-        mcg_obj,
-        cld_mgr,
-        awscli_pod,
-        bucket_factory,
-        bucketclass_dict,
-    ):
-        """
-        Test Write to ns bucket using MCG RPC and read directly from AWS.
-        """
-
-        ns_bucket = bucket_factory(
-            amount=1,
-            interface=bucketclass_dict["interface"],
-            bucketclass=bucketclass_dict,
-        )[0]
-
-        s3_creds = {
-            "access_key_id": cld_mgr.aws_client.access_key,
-            "access_key": cld_mgr.aws_client.secret_key,
-            "endpoint": constants.MCG_NS_AWS_ENDPOINT,
-            "region": self.DEFAULT_REGION,
-        }
-        aws_target_bucket = ns_bucket.bucketclass.namespacestores[0].uls_name
-
-        # Upload files to NS bucket
-        self.write_files_to_pod_and_upload(
-            mcg_obj, awscli_pod, bucket_to_write=ns_bucket.name, amount=3
-        )
-        # Read files directly from AWS
-        self.download_files(
-            mcg_obj, awscli_pod, bucket_to_read=aws_target_bucket, s3_creds=s3_creds
-        )
-
-        # Compare between uploaded files and downloaded files
-        assert self.compare_dirs(awscli_pod, amount=3)
-
     @pytest.mark.polarion_id("OCS-2255")
     @tier1
-    def test_namespace_resource_creation(self, ns_resource_factory):
+    @skipif_ocs_version(">4.6")
```

Comment:
Any reason we'd like to skip those above 4.6?
I don't think it's bad to cover both RPC and CRD flows - it's free extra coverage :)

Reply:
I think that we should not include them because we test with CRDs the exact same cases and because running all those tests again for NooBaa RPC calls can be too time consuming. CRDs are also preferred way how to manage bucket resources since 4.7.
@ebenahar @parikshithb WDYT?

Reply:
I think the same way as Filip.  Because in later releases, there might be some major/minor changes to these API calls which might cause failures and should be analyzed and updated which is time consuming.  Apart for that, IMO, most or all the users will use either  CRDs or UI(4.8) for this feature so maintaining these tests as well seem redundant to me.

Reply:
I don't see a reason why we want to test both CRDs and RPC with OCS > 4.6. IIUC, RPC was an intermediate API for NS bucket and now that we have CRDs, it should not be in use by customers